swift/looker/Tests/lookerTests/methodsTests.swift

```diff
@@ -85,78 +85,94 @@ class methodsTests: XCTestCase {
         } else {
             XCTAssertTrue(false, "Couldn't cast data from jsonData")
         }
+                
+        } catch {
+            print(error)
```

Comment:
Might even throw it to fail the test? Not sure what would happen here if it failed.

Reply:
I think I'll leave it with the current pattern for now. Still learning Swift practices.